demosplan/DemosPlanCoreBundle/Logic/Statement/DraftStatementService.php

```diff
@@ -175,7 +175,10 @@ public function getDraftStatementList($procedureId, $scope, StatementListUserFil
             throw new AccessDeniedException('No user given');
         }
         try {
-            // die eigenen Freigaben müssen abweichend vom Standardverfahren aus der Datenbank gezogen werden
+            // the released draftstatements have to be fetched differently from the database
+            // - do not use the default strategy in this case
+            // 'own' in combination with getReleased means the draftstatement belongs to the users orga
+            // and has been created by the given user.
```

Comment:
This is nitpicking, im sorry:

```suggestion
             /**
             * Special fetching strategy for DraftStatements is needed:
             * 'own' in combination with getReleased to indicate
             * the DraftStatement belongs to the users organisation and has been created by the given user.
             */
```